sageintacctsdk/apis/accounts.py

```diff
@@ -0,0 +1,61 @@
+"""
+Sage Intacct accounts
+"""
+from typing import Dict
+
+from .api_base import ApiBase
+
+
+class Accounts(ApiBase):
+    """Class for Accounts APIs."""
+
+    def post(self, data: Dict):
+        """Post general ledger account to Sage Intacct.
+
+        Returns:
+            Dict of state of request with RECORDNO.
+        """
+        data = {
+            'create': {
+                'GLACCOUNT': data
+            }
+        }
+        return self._format_post_request(data)
+
+    def get(self, field: str, value: str):
+        """Get general ledger account from Sage Intacct
+
+        Parameters:
+            field (str): A parameter to filter general ledger account by the field. (required).
+            value (str): A parameter to filter general ledger account by the field - value. (required).
+
+        Returns:
+            Dict in Location schema.
+        """
+        data = {
+            'readByQuery': {
+                'object': 'GLACCOUNT',
+                'fields': '*',
+                'query': "{0} = '{1}'".format(field, value),
+                'pagesize': '1000'
+            }
+        }
+
+        return self._format_post_request(data)['data']
```

Comment:
Why is this function called `format_post_request` If we're using this for both GET and POST
rather rename it to `format_request`